src/server/services/http/httpError.fixture.ts

```diff
@@ -0,0 +1,7 @@
+import { AxiosResponse } from 'axios';
+
+import { HttpError } from '~/server/services/http/httpError';
+
+export function anHttpError(status?: number, message?: string, response?: AxiosResponse): HttpError {
+	return new HttpError(status || 500, message || 'An error occurred', response);
+}
```

Comment:
Pour permettre d'alléger le code des tests et ne pas instancier à la main une erreur axios avec des valeurs non testées (mais qui sont utilisés dans le code), on pourrait avoir l'AxiosResponse en valeur par défaut également :

```suggestion
export function anHttpError(status?: number = 500, message?: string = 'An error occurred', providedResponse?: AxiosResponse): HttpError {
         const response = providedResponse ?? anAxiosResponse(
					{
						message,
					},
					status,
				)) 
	return new HttpError(status, message, response);
}
```